qa/restart-upgrade/build.gradle

```diff
@@ -0,0 +1,87 @@
+/*
+ *  Copyright OpenSearch Contributors
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import org.opensearch.gradle.testclusters.StandaloneRestIntegTestTask
+import org.apache.tools.ant.taskdefs.condition.Os
+import java.util.concurrent.Callable
+
+apply from : "$rootDir/qa/build.gradle"
+
+String default_bwc_version = System.getProperty("bwc.version")
+String neural_search_bwc_version = System.getProperty("tests.bwc.version", default_bwc_version)
+boolean isSnapshot = neural_search_bwc_version.contains("-SNAPSHOT")
```

Comment:
Lets stick to `snake_case` for var names.

Reply:
Can you point to the variable you are referring to?